### PR TITLE
AG-13250 - update icon mappings

### PR DESCRIPTION
**Pull request by peterjrreynolds**

### CODE REVIEW

File: packages/ag-grid-community/src/headerRendering/cells/headerModule.ts [hunk 1]
```diff
@@ -8,11 +8,23 @@ export const ColumnHeaderCompModule: _ModuleWithoutApi = {
...
 export const ColumnGroupHeaderCompModule: _ModuleWithoutApi = {
     ...baseCommunityModule('ColumnGroupHeaderCompModule'),
     userComponents: {
         agColumnGroupHeader: HeaderGroupComp,
     },
+    icons: {
+        // header column group shown when expanded (click to contract)
+        columnGroupOpened: 'expanded',
+        // header column group shown when contracted (click to expand)
+        columnGroupClosed: 'contracted',
```
Comment (StephenCooper) [inline packages/ag-grid-community/src/headerRendering/cells/headerModule.ts:28]: If we as devs forget to provide an icon in the module will it be obvious what we need to do?

File: packages/ag-grid-community/src/validation/errorMessages/errorText.ts [hunk 2]
```diff
@@ -12,20 +12,32 @@ const missingModule = ({
...
-    moduleName: ModuleName;
+    moduleName: ModuleName | ModuleName[];
     gridScoped: boolean;
     gridId: string;
     isEnterprise?: boolean;
-}) =>
-    `Unable to use ${reason} as ${moduleName} is not registered${gridScoped ? ' for gridId: ' + gridId : ''}. Check if you have registered the module:
+    additionalText?: string;
+}) => {
+    // TODO - update to list multiple
```
Comment (StephenCooper) [inline packages/ag-grid-community/src/validation/errorMessages/errorText.ts:24]: Old TODO comment?

Comment (StephenCooper) [inline packages/ag-grid-community/src/validation/errorMessages/errorText.ts:24]: Guessing this will be done as part of module validation work?

**Review by StephenCooper — APPROVED** [review]

File: packages/ag-grid-community/src/validation/errorMessages/errorText.ts (hunk 2 above)
Comment (peterjrreynolds) [reply packages/ag-grid-community/src/validation/errorMessages/errorText.ts:24]: Yes

File: packages/ag-grid-community/src/headerRendering/cells/headerModule.ts (hunk 1 above)
Comment (peterjrreynolds) [reply packages/ag-grid-community/src/headerRendering/cells/headerModule.ts:28]: If the validation module is included, there will be a warning

**Pull request merged**